### ISSUE TITLE
Even after shutting down computer, Brave is keeping logged in to sites

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#7697]
**Describe the issue you encountered:**

https://community.brave.com/t/even-after-shutting-down-computer-brave-is-keeping-me-logged-in-to-everything/1686?u=suguru

<blockquote>
Settings: 
Shields up. 
Clear the following data types when I close Brave: everything is checked.
Do not track: checked.
Password Manager: don't manage my passwords.

So how is it that when I restart my computer and I start Brave back up again, I find that I'm still logged in to many sites that I use in the operation of my business?
</blockquote>

- Platform (Win7, 8, 10? macOS? Linux distro?):

- Brave Version (revision SHA): 0.13.5

- Steps to reproduce:
    1.
    2.
    3.
    
- Actual result:

- Expected result:

- Will the steps above reproduce in a fresh profile? If not what other info can be added?

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by bbondy** [comment]
I don't think I understand, cookies will be stored for the site to save login information. You can control to block cookies from a site via the Brave Shield though.  Or change the global options.

Let me know if I misunderstood the issue, please feel free to re-open if that's the case.

**Comment by luixxiul** [comment]
I assume the feedback claims that the cookies are not cleared by closing the browser though the option to clear them is enabled on about:preferences

**Comment by Jacalz** [comment]
Yes the user have the option set to delete everything in the settings but they are not deleted.

**Comment by bsclifton** [comment]
@Jacalz are you experiencing this issue also? Or if you haven't, are you able to reproduce?

**Comment by Jacalz** [comment]
No I can't reproduce and I am not having the issue, just opening since the user from the community where still having the problem 😉 @bsclifton

**Comment by Liunkae** [comment]
Does Chromium have some kind of cache? Some things are not cleared (dev tool settings, visited links #9351). This could be caused by that.

**Comment by diracdeltas** [comment]
@Liunkae yeah, i believe so. we should have a `Reset profile` button which just deletes your browser profile folder.
